constellations/gemini-user/src/afh/afh.js

```diff
@@ -0,0 +1,152 @@
+                </div>
+            );
+        });
+
+        return (
+            <div id="view-afh">
+                <h1>Ask for Help</h1>
+                <div className="description">
+                    We provide free sessions for students to ask for additional
+                    assistance on any of our program subjects. Please fill the
+                    form to let us know you are coming. You must be registered
+                    with one of our programs to attend.
+                </div>
+
+                <h1>Ask for Help Sessions by Subject</h1>
+                <div className="tab">
+                    <button
+                        className={
+                            this.state.currentTab == "Math" ? "active" : ""
+                        }
+                        onClick={() => this.openSubject("Math")}>
+                        Math
+                    </button>
+                    <button
+                        className={
+                            this.state.currentTab == "English" ? "active" : ""
+                        }
+                        onClick={() => this.openSubject("English")}>
+                        English
+                    </button>
+                    <button
+                        className={
+                            this.state.currentTab == "Computer Programming"
+                                ? "active"
+                                : ""
+                        }
+                        onClick={() =>
+                            this.openSubject("Computer Programming")
+                        }>
+                        Computer Programming
+                    </button>
+                </div>
+
+                <div
+                    className={
+                        this.state.currentTab == "Math" ? "showTab" : "hide"
+                    }>
+                    {showSessions}
+                </div>
+
+                <div
+                    className={
+                        this.state.currentTab == "English" ? "showTab" : "hide"
+                    }>
+                    {showSessions}
+                </div>
+
+                <div
+                    className={
+                        this.state.currentTab == "Computer Programming"
+                            ? "showTab"
+                            : "hide"
+                    }>
+                    {showSessions}
+                </div>
+            </div>
+        );
+    }
+}
+
+class CheckboxInput extends React.Component {
+    state = {
+        isActive: "false",
+    };
+    selectCheckbox = (e) => {
+        return <input className="select" type="checkbox" />;
+    };
+
+    onSelectSession = (e) => {
+        if (this.state.isActive == "false") {
+            this.setState({
+                isActive: "true",
+            });
+        } else {
+            this.setState({
+                isActive: "false",
+            });
+        }
+        return this.state.isActive;
```

Comment:
I don't think you need to return anything here. It doesn't look like you're doing anything with what you return?